src/cloudsc_pyiface/fortransrc/cloudsc.F90

```diff
@@ -505,11 +505,11 @@ SUBROUTINE CLOUDSC &
 #include "fcttre.ycst.h"
 #include "fccld.ydthf.h"
 
+#include "micvarsnumbs.h"
```

Comment:
I'm not sure this is what @mlange05 had in mind. `NCLDQV`, `NCLDQL`, `NCLDQR`, `NCLDQI`, `NCLDQS` are already declared as parameters in yoecldp.F90, why can't they be imported directly, in the same way as JPRB & co?

Reply:
Yeah, I fail to see how moving them to a new .h file solves any of this, as the Python is not touching this either. In any case, cleanly using existing modules works just as well (I confirmed locally that there's no performance impact), so I'll revert to using only the common module for the constants, and the performance impact analysis can be done in a follow-up.

Reply:
If I understand correctly, including .h means that the parameter live in the same program unit as the code that uses them, whereas when importing from a module means importing from a "common" library. Overall, however, no objections.